src/Action/CaptureAction.php

```diff
@@ -200,15 +196,24 @@ private function addNotificationUrl(TokenInterface $token, ArrayObject $details)
         $notifyToken = $this->tokenFactory->createNotifyToken($token->getGatewayName(), $token->getDetails());
 
         $notificationUrl = $notifyToken->getTargetUrl();
-
         $details['notification_url'] = $notificationUrl;
 
         return $details;
     }
 
-    private function createPayment(ArrayObject $details): Payment
+    /**
+     * @throws UnknownApiErrorException
+     * @throws BadRequestException
+     */
+    private function createPayment(ArrayObject $details, Capture $request): Payment
     {
         try {
+            if ($details->offsetExists('payment_id')
+                && $details->offsetExists('status')) {
+                $this->abortPaymentProcessor->process($request->getFirstModel());
+                unset($details['status'], $details['payment_id'], $details['is_live']);
```

Comment:
Why did you unset 'is_live' ?

Reply:
payplug is the data master for these fields, so we just update them using their response after the payment creation

Reply:
Ok, i understand.